src/rime/gear/key_binder.cc

```diff
@@ -94,7 +93,7 @@ static void select_schema(Engine* engine, const string& schema) {
   }
 }
 
-void KeyBindings::LoadBindings(const an<ConfigList>& bindings) {
+void KeyBindings::LoadBindings(const an<ConfigList>& bindings, bool bind_sequence) {
```

Comment:
`bind_sequence` applies to all `KeyBinding` entries. This is not ideal because it's an error including the preset single key bindings while adding key-sequence bindings.
Please make this configurable in individual `KeyBinding` entries: 
`send: comma` binds to a single `,` key and `send_sequence: comma` sends the word "comma".